File: src/Atlasd/Battlenet/Protocols/MCP/Messages.cs
namespace Atlasd.Battlenet.Protocols.MCP
{
    enum Messages
    {
        MCP_STARTUP = 0x01,
        MCP_CHARCREATE = 0x02,
        MCP_CREATEGAME = 0x03,
        MCP_JOINGAME = 0x04,
        MCP_GAMELIST = 0x05,
        MCP_GAMEINFO = 0x06,
        MCP_CHARLOGON = 0x07,
        MCP_CHARDELETE = 0x0A,
        MCP_REQUESTLADDERDATA = 0x11,
        MCP_MOTD = 0x12,
        MCP_CANCELGAMECREATE = 0x13,
        MCP_CREATEQUEUE = 0x14,
        MCP_CHARRANK = 0x16,
        MCP_CHARLIST = 0x17,
        MCP_CHARUPGRADE = 0x18,
        MCP_CHARLIST2 = 0x19,
    }
}

File: src/AtlasV/Battlenet/Protocols/MCP/Messages.vb
Namespace AtlasV.Battlenet.Protocols.MCP
    Enum Messages As Byte
        MCP_STARTUP = &H1
        MCP_CHARCREATE = &H2
        MCP_CREATEGAME = &H3
        MCP_JOINGAME = &H4
        MCP_GAMELIST = &H5
        MCP_GAMEINFO = &H6
        MCP_CHARLOGON = &H7
        MCP_CHARDELETE = &HA
        MCP_REQUESTLADDERDATA = &H11
        MCP_MOTD = &H12
        MCP_CANCELGAMECREATE = &H13
        MCP_CREATEQUEUE = &H14
        MCP_CHARRANK = &H16
        MCP_CHARLIST = &H17
        MCP_CHARUPGRADE = &H18
        MCP_CHARLIST2 = &H19
    End Enum
End Namespace